src/pages/index.js

```diff
@@ -83,32 +145,37 @@ class CV extends React.Component {
   render() {
     return (
       <div>
-          <CVPage firstPage={true}
-                  headerinfo={this.state.headerinfo}
-                  extraPages={this.state.extraPages}
-                  workexp={this.state.workexp}
-                  education={this.state.education}
-                  lists={this.state.lists}
-                  ref={this.childRef}
-                  showHeight={this.showHeightInConsole}
-                  checkBool={this.changeBool}
-                  addPage={this.addPage}
-                  />
-
-        {this.state.pages.map((el) => {
+        <CVPage
```

Comment:
Ik denk dat het waardevol is om een eslint/prettier toe te voegen om dit soort changes te voorkomen

Reply:
Ik heb prettier en eslint als extensie in vs code. En er zitten wel prettier files in dit project. Moet ik nog een eslintrc.js file toevoegen?